pkg/operator/ceph/object/admin.go

```diff
@@ -209,6 +209,10 @@ func RunAdminCommandNoMultisite(c *Context, expectJSON bool, args ...string) (st
 	var output, stderr string
 	var err error
 
+	// run radosgw-admin command with debug flags if Rook Operator log level is DEBUG
+	if logger.LevelAt(capnslog.DEBUG) {
```

Comment:
@thotz Discussed in huddle today... Running the ceph daemons at a higher log level can sometimes cause the commands to fail if unexpected output is produced so we will really need to test with this enabled to ensure nothing is broken. As the first step, have you tested some rgw scenarios locally with this enabled?

Reply:
It's convenient to change log levels of operators and other ceph commands. How about adding new configuration in `rook-ceph-operator-config` to toggle the latter commands?

Reply:
@travisn : The user creation command for radosgw-admin failed with an invalid flag error here, I check that issue. Secondly, even with failure, the log prints the err value, not the stdout of the command. So I am wondering whether this PR is needed or not. It does not make any difference. Or is there any log collect the stdout o/p?

Reply:
We expect to capture all the output from the command. 
@leseb Any ideas on why the output might not be captured in this case?

Reply:
It's hard to say without a log trace.

Reply:
@leseb following are the logs with my PR(one success and failure commands) :
```
 | exec: Running command: radosgw-admin realm get --rgw-realm=my-store --debug-rgw 20 --debug-rgw_sync 20 --debug-ms 20 --cluster=rook-ceph --conf=/var/lib/rook/rook-ceph/rook-ceph.config --name=client.admin --keyring=/var/lib/rook/rook-ceph/client.admin.keyring
2021-10-01 06:51:47.306631 I | ceph-object-controller: multisite configuration for object-store my-store is complete
2021-10-01 06:51:47.306708 I | ceph-object-controller: creating object store "my-store" in namespace "rook-ceph"
2021-10-01 06:51:47.306730 I | cephclient: getting or creating ceph auth key "client.rgw.my.store.a"
2021-10-01 06:51:47.306777 D | exec: Running command: ceph auth get-or-create-key client.rgw.my.store.

2021-10-01 06:51:49.768634 D | exec: Running command: radosgw-admin user create --uid dashboard-admin --display-name dashboard-admin --system --yes-i-really-mean-it --rgw-realm= --rgw-zonegroup= --rgw-zone= --debug-rgw 20 --debug-rgw_sync 20 --debug-ms 20 --cluster=rook-ceph --conf=/var/lib/rook/rook-ceph/rook-ceph.config --name=client.admin --keyring=/var/lib/rook/rook-ceph/client.admin.keyring
2021-10-01 06:51:49.811014 W | ceph-object-controller: failed to enable dashboard for rgw. failed to create user "dashboard-admin": failed to create s3 user. ERROR: invalid flag --debug-rgw 20 --debug-rgw_sync 20 --debug-ms 20. : exit status 22
2021-10-01 06:51:49.811162 I | ceph-object-controller: created object store "my-store" in namespace "rook-ceph"
2021-10-01 06:51:49.811191 D | ceph-object-controller: creating s3 user object "rgw-admin-ops-user" for object store "rook-ceph"
2021-10-01 06:51:49.811276 D | ceph-object-controller: creating s3 user "rgw-admin-ops-user"
2021-10-01 06:51:49.811307 D | exec: Running command: radosgw-admin user create --uid rgw-admin-ops-user --display-name RGW Admin Ops User --caps buckets=*;users=*;usage=read;metadata=read;zone=read --rgw-realm=my-store --rgw-zonegroup=my-store --rgw-zone=my-store --debug-rgw 20 --debug-rgw_sync 20 --debug-ms 20 --cluster=rook-ceph --conf=/var/lib/rook/rook-ceph/rook-ceph.config --name=client.admin --keyring=/var/lib/rook/rook-ceph/client.admin.keyring
me=client.admin --keyring=/var/lib/rook/rook-ceph/client.admin.keyring
2021-10-01 06:51:49.855412 D | ceph-spec: update event from a CR
2021-10-01 06:51:49.855441 D | ceph-spec: update event on CephObjectStore CR
2021-10-01 06:51:49.855555 D | clusterdisruption-controller: reconciling "rook-ceph/"
2021-10-01 06:51:49.856060 D | ceph-object-controller: object store "rook-ceph/my-store" status updated to "Failure"
2021-10-01 06:51:49.856093 E | ceph-object-controller: failed to reconcile CephObjectStore "rook-ceph/my-store". failed to create object store deployments: failed to start rgw health checker for CephObjectStore "rook-ceph/my-store", will re-reconcile: failed to create bucket checker for CephObjectStore "rook-ceph/my-store": failed to create or retrieve rgw admin ops user: failed to create object user "rgw-admin-ops-user". error code 1 for object store "my-store": failed to create s3 user. ERROR: invalid flag --debug-rgw 20 --debug-rgw_sync 20 --debug-ms 20. : exit status 22
```

Reply:
You need to address my previous comment, the CLI args is incorrect. It's treated as one blob by bash and we need separate arguments. See https://github.com/rook/rook/pull/8836/files#r716554453